src/styles/styles.scss

```diff
@@ -24,3 +24,4 @@
 [id*='lastupdated-'] {
   display: none;
 }
+@import './link-card.css';
```

Comment:
Please place this above organized with the other primitives in alphabetical order